src/runtime/engine/execute_process_service.ts

```diff
@@ -253,6 +254,27 @@ export class ExecuteProcessService implements IExecuteProcessService {
 
     await this._saveCorrelation(identity, processInstanceConfig.correlationId, processInstanceConfig.processInstanceId, processModel.id);
 
+    const processStartedMessage: ProcessStartedMessage = new ProcessStartedMessage(processInstanceConfig.correlationId,
+      processModel.id,
+      processInstanceConfig.processInstanceId,
+      processInstanceConfig.startEvent.id,
+      null, processInstanceConfig.processToken);
```

Comment:
```suggestion
      null,
      processInstanceConfig.processToken);
```

What does `null` stand for?!

Reply:
you could give it a name above it, like:

```ts

const xyz = null;

(...)
xyz,
processInstanceConfig.processToken);
(...)

Reply:
The `flowNodeInstanceId` is not known when the MessageObject is created. Thats why I set this parameter to `null`.

Reply:
See my suggestion here:
https://github.com/process-engine/process_engine_contracts/pull/84#discussion_r235693289

Reply:
I would still change it to something like
```
    const flowNodeInstanceId: string = null;

    const processStartedMessage: ProcessStartedMessage = new ProcessStartedMessage(processInstanceConfig.correlationId,
      processModel.id,
      processInstanceConfig.processInstanceId,
      processInstanceConfig.startEvent.id,
      flowNodeInstanceId,
      processInstanceConfig.processToken);
```

Reply:
@SteffenKn see https://github.com/process-engine/process_engine_contracts/pull/84#discussion_r235693289 
This shouldn't be an issue anymore.